oracle.py

```diff
@@ -208,4 +296,55 @@ def check(learned_side_effect, learned_status, testing_side_effect, testing_stat
                 testing_config)
         bug_report = "\n[BUG REPORT]\n" + bug_report
         print(bug_report)
+
+    if learned_resources != None:
+        look_for_resouces_diff(learned_resources, testing_resources)
```

Comment:
I guess we should generate bug report according to `look_for_resources_diff`, right?

Reply:
and also generate alarm